settings.py

```diff
@@ -7,82 +8,109 @@
 #     https://docs.scrapy.org/en/latest/topics/downloader-middleware.html
 #     https://docs.scrapy.org/en/latest/topics/spider-middleware.html
 
+# USER_AGENT = get_random_agent()
+
 BOT_NAME = 'scraping'
 
 SPIDER_MODULES = ['scraping.spiders']
 NEWSPIDER_MODULE = 'scraping.spiders'
 
-
 # Crawl responsibly by identifying yourself (and your website) on the user-agent
-#USER_AGENT = 'scraping (+http://www.yourdomain.com)'
+# USER_AGENT = 'scraping (+http://www.yourdomain.com)'
 
 # Obey robots.txt rules
 ROBOTSTXT_OBEY = False
 
+# HTTPERROR_ALLOWED_CODES = [404]
+# USER_AGENT = 'quotesbot (+http://www.yourdomain.com)'
+# USER_AGENT = "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/34.0.1847.131 Safari/537.36"
+
+
+# PROXY_POOL_ENABLED = True
+
 # Configure maximum concurrent requests performed by Scrapy (default: 16)
-#CONCURRENT_REQUESTS = 32
+# CONCURRENT_REQUESTS = 32
```

Comment:
Why all the settings are commented out ?

Reply:
before i was testing in same file